src/Components/Web/src/Forms/InputRadio.cs

```diff
@@ -11,58 +12,58 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// <summary>
     /// An input component used for selecting a value from a group of choices.
     /// </summary>
-    public class InputRadio<TValue> : InputBase<TValue>
+    public class InputRadio<TValue> : ComponentBase
     {
         /// <summary>
-        /// Gets the name of this <see cref="InputRadio{TValue}"/> group.
+        /// Gets context for this <see cref="InputRadio{TValue}"/>.
         /// </summary>
-        protected string? GroupName { get; private set; }
+        protected InputRadioContext? Context { get; private set; }
```

Comment:
I'm curious about why this is `protected`. Do you anticipate use cases for subclassing `InputRadio` and accessing `Context`?

It would be pretty cool if `InputRadioContext` were able to be an internal type.

Reply:
I made the context `protected` mostly so the tests were able to use the context property to verify that an `InputRadio` group name was correct. If I make the context `private` and change `InputRadioContext` to internal, do you think it would be satisfactory to expose the group name separately?

Reply:
Ah right. If this is just for tests, you can mark it as `internal` and then add an `InternalsVisibleTo` entry in [here](https://github.com/dotnet/aspnetcore/blob/master/src/Components/Web/src/Properties/AssemblyInfo.cs) so you can access it from Microsoft.AspNet.Components.Web.Tests. We do this a lot for tests :)